samples/helpers/get-env/inc/get_env.hpp

```diff
@@ -34,18 +34,3 @@ char* getenv(const char* name);
 #endif
 
 #endif
-#include <stdexcept>
```

Comment:
There is a fairly significant behavior difference between `GetEnvHelper::GetEnv` and `std::getenv` which means that one is not a plugin replacement for the other.

`GetEnvHelper::GetEnv` always returns a valid `std::string`, but `std::getenv` does not. That means that if the environment variables are not defined, the sample will crash (the `std::string` constructor crashes if the value passed in is null), but `GetEnvHelper::GetEnv` will emit a diagnostic message to the developer if the environment variable is not present.